src/app/modules/submission/encrypt-submission/encrypt-submission.controller.ts

```diff
@@ -432,6 +433,13 @@ const submitEncryptModeForm: ControllerHandler<
       },
     })
 
+    // TODO 6395 make responseMetadata mandatory
+    if (responseMetadata) {
+      reportSubmissionResponseTime(responseMetadata, {
+        mode: 'encrypt',
+        payment: false as unknown as string,
```

Comment:
LOL... No need to lie on types for this 😅 , you can just go for:
```suggestion
        payment: 'false'
```

or `'0'` also can 😄

Reply:
HAHHAAH okay have reverted it back to string 😅 [bf8343a](https://github.com/opengovsg/FormSG/pull/6406/commits/bf8343a7ddb8bab2c8c17d2a6a37c1609c69f532)